raiden/blockchain/events.py

```diff
@@ -389,6 +390,23 @@ def add_netting_channel_listener(
             netting_channel_proxy.all_events_filter,
         )
 
+    def add_payment_channel_listener(
+        self,
+        payment_channel_proxy: PaymentChannel,
+        from_block: typing.BlockSpecification = 'latest',
+    ):
+        payment_channel_proxy.all_events_filter(from_block=from_block)
+        channel_identifier = payment_channel_proxy.channel_identifier()
+
+        self.add_event_listener(
+            f'PaymentChannel event {channel_identifier}',
+            payment_channel_proxy,
```

Comment:
this should be the filter and not the proxy

Reply:
Yeah, don't know what I did there.